client/__tests__/Comparison.test.js

```diff
@@ -0,0 +1,173 @@
+        'url': 'https://images.unsplash.com/photo-1422728221357-57980993ea99?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80'
+      },
+      {
+        'thumbnail_url': 'https://images.unsplash.com/photo-1488778578932-0f84d315fcae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+        'url': 'https://images.unsplash.com/photo-1488778578932-0f84d315fcae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=658&q=80'
+      },
+      {
+        'thumbnail_url': 'https://images.unsplash.com/photo-1527431016-15eb83515018?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+        'url': 'https://images.unsplash.com/photo-1527431016-15eb83515018?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80'
+      },
+      {
+        'thumbnail_url': 'https://images.unsplash.com/photo-1534550017194-5df79ed78967?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+        'url': 'https://images.unsplash.com/photo-1534550017194-5df79ed78967?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80'
+      },
+      {
+        'thumbnail_url': 'https://images.unsplash.com/photo-1525896650794-60ad4ec40d0e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+        'url': 'https://images.unsplash.com/photo-1525896650794-60ad4ec40d0e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80'
+      },
+      {
+        'thumbnail_url': 'https://images.unsplash.com/photo-1560857792-215f9e3534ed?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+        'url': 'https://images.unsplash.com/photo-1560857792-215f9e3534ed?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80'
+      }
+    ],
+    'skus': {
+      '714558': {
+        'quantity': 14,
+        'size': '7'
+      },
+      '714559': {
+        'quantity': 25,
+        'size': '7.5'
+      },
+      '714560': {
+        'quantity': 9,
+        'size': '8'
+      },
+      '714561': {
+        'quantity': 2,
+        'size': '8.5'
+      },
+      '714562': {
+        'quantity': 18,
+        'size': '9'
+      },
+      '714563': {
+        'quantity': 12,
+        'size': '9.5'
+      },
+      '714564': {
+        'quantity': 10,
+        'size': '10'
+      },
+      '714565': {
+        'quantity': 18,
+        'size': '10.5'
+      },
+      '714566': {
+        'quantity': 11,
+        'size': '11'
+      },
+      '714567': {
+        'quantity': 35,
+        'size': '11.5'
+      },
+      '714568': {
+        'quantity': 25,
+        'size': '12'
+      }
+    }
+  }
+};
+
+const relatedProductInformation = JSON.parse(JSON.stringify(currentProductInformation));
+relatedProductInformation.features.push({
+  feature: 'this feature is only on the related product',
+  value: true
+});
```

Comment:
In an ideal world, I'd also remove a feature that was in the original, just so regression tests in the future will make sure that doesn't cause crashes. But that's a pretty small nit to pick.

Reply:
This is a good idea, thank you. I'll add this as a ticket